libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/listeners/SpeechAnnouncementListener.java

```diff
@@ -19,8 +19,8 @@
    * and it will be ignored.
    *
    * @param announcement about to be announced
-   * @return announcement to be played; null if should be ignored
-   * @since 0.16.0
+   * @return text announcement to be played; null if should be ignored
+   * @since 0.18.0
    */
-  SpeechAnnouncement willVoice(SpeechAnnouncement announcement);
+  String willVoice(SpeechAnnouncement announcement);
```

Comment:
👍 This is a good idea, in the case of speech announcements which are simpler.

Reply:
Yeah, as you mentioned voice instructions are simpler because basically there's only one option to override - the announcement that you want the speech engine to speak. To be concrete here I want to note that we're removing the `ssmlAnnouncement` - which I don't think that any developer would want to take advantage of but as said, if the time comes we can revisit.